fabtests/include/windows/osd.h

```diff
@@ -137,6 +136,26 @@ int socketpair(int af, int type, int protocol, int socks[2]);
 #define WCONTINUED	8	/* Report continued child. */
 #define WNOWAIT		0x01000000	/* Don't reap, just poll status. */
 
+#define MSG_DONTWAIT	(1 << 6)	/* Equivalent to unix MSG_DONTWAIT */
```

Comment:
Do we know that this won't conflict with a windows defined socket flag?

Reply:
Windows defines the following flags for usage in the socket calls:
#define MSG_OOB         0x1             /* process out-of-band data */
#define MSG_PEEK        0x2             /* peek at incoming message */
#define MSG_DONTROUTE   0x4     /* send without using routing tables 
#define MSG_WAITALL     0x8          /* do not complete until packet is

I picked 1 << 6 (0x40) to match the Linux definitions but I can pick a more Windows-ish value if you think that makes more sense?

Reply:
We need to ensure that we're not picking an existing windows value.  I don't know how we easily figure that out.